packages/plugins/src/foundry/utils/index.ts

```diff
@@ -1086,11 +1107,18 @@ export const getEstimatedGas = async (
   })
   const adjustedGasLimit = Number(estimatedMinGasLimit) - 21_000
 
-  const estimatedGas = receipts
-    .map((receipt) => receipt.gasUsed)
-    .map(Number)
-    .map((gasUsed) => Math.round(gasUsed * 1.3))
-    .map((gasWithBuffer) => {
+  const transactionsWithGasEstimates = transactions
+    .map((transaction) => {
+      return {
+        transaction: toSphinxTransactionEstimatedGas(transaction.response),
+        estimatedGas: Math.round(Number(transaction.receipt.gasUsed) * 1.3),
+        estimatedBlobGas: Math.round(
+          Number(transaction.receipt.blobGasUsed ?? 0) * 1.3
+        ),
```

Comment:
I've included a 30% buffer on the blob gas for consistency. Let me know if you think there's a reason to do something else here.